techniques/general/G78.html

```diff
@@ -1,10 +1,9 @@
 <!DOCTYPE html><html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml"><head><title>Providing a second, user-selectable, audio track that includes audio descriptions</title><link rel="stylesheet" type="text/css" href="../../css/sources.css" class="remove"></link></head><body><h1>Providing a second, user-selectable, audio track that includes audio descriptions</h1><section class="meta"><p class="id">ID: G78</p><p class="technology">Technology: general</p><p class="type">Type: Technique</p></section><section id="applicability"><h2>When to Use</h2>
-      <p>Applies to any technology that has a sound track and visual content.</p>
+      <p>Applies to any technology that has a soundtrack and synchronized visual content.</p>
    </section><section id="description"><h2>Description</h2>
       <p>The objective of this technique is to provide an audio (spoken) version of information that is provided visually so that it is possible for people who cannot see to be able to understand audio-visual material.</p>
-      <p>Since most user agents today cannot merge multiple sound tracks, this technique adds the additional audio information to synchronized media by providing an option which allows users to replace the soundtrack with a new copy of the original soundtrack that has the additional audio description added. This added information focuses on actions, characters, scene changes and on-screen text (not captions) that are important to understanding the content.</p>
-      <p>Since it is not helpful to have this new information obscure key audio information in the original sound track (or be obscured by loud sound effects), the new information is added during pauses in dialogue and sound effects. This limits the amount of supplementary information that can be added to the program.</p>
-      <p>The soundtrack with the audio description (of visual information) can either be an alternate sound track that the user can choose, or it can be the standard sound track that everyone hears.</p>
```

Comment:
Seems like the text from earlier could be used instead of this deletion:

```
<p>A single movie with audio description in its single audio track can be presented to all users. However, audio description can be distracting for those who do not need it, so versions of the movie with and without audio description can be provided, with a mechanism for users to choose the version they prefer.</p>
```